lib/views/help/officers.html.erb

```diff
@@ -1,247 +1,195 @@
+
+      <p>The FOI request you received was made by someone using <%= site_name %>. You can
+        simply reply to the request as you would any other request from an individual.
+        The only difference is that your response will be automatically published on
+        the Internet.
+      </p>
+      <p>If you have privacy or other concerns, please read the answers below.
+        You might also like to read the <a
+        href="<%= help_about_path %>">introduction to <%= site_name %></a> to find out more about what
+        the site does from the point of view of a user. You can also search the
+        site to find the authority that you work for, and view the status of
+        any requests made using the site.
+
+      <p>Finally, we welcome comments and
+        thoughts from FOI officers, please <a href="<%= help_contact_path %>">get in touch</a>.
+      </p>
+    </dd>
+
+    <dt id="responses">Why are you publishing responses to FOI requests? <a href="#responses">#</a> </dt>
+
+    <dd>We think there are lots of benefits. Most importantly it will encourage the
+      public to be more interested and involved in the work of government. We
+      also hope that it will reduce the number of duplicate requests on any
+      subject that a public body will receive. Given that Freedom of Information
+      responses contain public information, which anybody could easily request
+      again from the public authority, we think there should be no reason not to
+      publish it widely.
+    </dd>
+
+    <dt id="realpeople">Are the people making requests real people? <a href="#realpeople">#</a> </dt>
+
+    <dd>Yes. For the purposes of keeping track of responses we use
+      computer-generated email addresses for each request. However, before
+      they can send a request, each user must register on the site with a
+      unique email address that we then verify. You can search this site and
+      find a page listing all requests that each person has made.
+    </dd>
+
+    <dt id="email_only">An email isn't a sufficient address for an FOI request! <a href="#email_only">#</a> </dt>
 
-<p>The Freedom of Information (Scotland) Act does not allow such a public
-interest extension. <%= site_name %> would like to see the law changed to either
-remove the extension from the UK Act, or to reintroduce an absolute time limit
-of 40 working days even with the extension (the House of Lords <a
-href="http://www.publicwhip.org.uk/division.php?date=2000-10-17&amp;number=1&amp;house=lords">voted
-to remove</a> provision for such a time limit during the initial passage
-of the UK Act through Parliament).
-</p>
-</dd>
+    <dd>Yes it is. In the UK, this
+      <a href="https://www.whatdotheyknow.com/request/1142/response/2894/attach/5/20080806100741260.pdf">letter from the ICO to Rother District Council</a> gives guidance on the matter, specifically
+      in the context of requests made via WhatDoTheyKnow, a similar site to <%= site_name %>.
```

Comment:
would I be worrying unnecessarily that this might produce the text "requests made via WhatDoTheyKnow, a similar site to WhatDoTheyKnow."?